dogfooding/lib/app/app_content.dart

```diff
@@ -120,6 +121,19 @@ class _StreamDogFoodingAppContentState
     _observeFcmMessages();
   }
 
+  void _handleMobileAudioInterruptions() {
+    if (!CurrentPlatform.isMobile) return;
+
+    RtcMediaDeviceNotifier.instance.handleCallInterruptionCallbacks(
+      onInterruptionStart: () {
+        StreamVideo.instance.activeCall?.setMicrophoneEnabled(enabled: false);
+      },
+      onInterruptionEnd: () {
+        StreamVideo.instance.activeCall?.setMicrophoneEnabled(enabled: true);
+      },
+    );
+  }
```

Comment:
I do think it's good to keep track of the previous state, otherwise it will enable it when it was disabled before.

Maybe also good to mention in the docs that you have to loop through all `activeCalls` if you `allowMultipleActiveCalls`.

```suggestion
  bool? _microphoneEnabledBeforeInterruption;
  
  void _handleMobileAudioInterruptions() {
    if (!CurrentPlatform.isMobile) return;

    RtcMediaDeviceNotifier.instance.handleCallInterruptionCallbacks(
      onInterruptionStart: () {
        final call = StreamVideo.instance.activeCall;
        call?.setMicrophoneEnabled(enabled: false);
        _microphoneEnabledBeforeInterruption = call?.state.localParticipant?.isAudioEnabled;
      },
      onInterruptionEnd: () {
        if(_microphoneEnabledBeforeInterruption == true){
          StreamVideo.instance.activeCall?.setMicrophoneEnabled(enabled: true);
        }
        _microphoneEnabledBeforeInterruption = null;
      },
    );
  }
```